### ISSUE TITLE
Add a --version option

### ISSUE THREAD

**Issue by tseemann** [xflouris/PEAR#29]
Can you please add a GNU standard style `--version` option to aid software auditing?

```
% pear --version
pear 0.9.6
```

Thank you.

**Comment by xflouris** [comment]
Hi @tseemann , I've added a --version option (shortcut is -x since -v is taken already). I'm planning to have all short options removed though, and keep only long options in the new re-designed pear.

**Comment by marcelm** [comment]
Thanks for the `--version` option, I’m trying to package PEAR for the bioconda channel and was just looking for that option. However, could you change it such that the program then exits with an error code of zero? I need some way to run the program successfully, preferably without actually merging any reads. And while you are at it, it would be great if running `pear --help` would also return with exit code 0 (that’s what most other programs do).

**Comment by tseemann** [comment]
Yes, `--version` and `--help` should return exit code 0 because the completed successfully.
